example.py

```diff
@@ -0,0 +1,20 @@
+# example for using dart_canvas_parser package
+from collections import namedtuple
+from dart_canvas_parser.parser import CanvasParser
+from dart_sdk.models.canvas_content_source import CanvasContentSource
+
+import secure
+
+content_source = CanvasContentSource(
+    default_license = {
```

Comment:
Yes, it's annoying that we have to do this. I wonder if, for the sake of just using the data model to parse things, we could subclass `dart_sdk.models.canvas_content_source.CanvasContentSource` (and its parent, `ContentSource`) and remove the requirements we don't need. You could imagine telling the parser to use the canonical `CanvasContentSource` in the event it's being called from DART, or a modified version if used for research/export. I think the problem there is that we'd really love to retain information about the provenance of the conversion, which gets captured in some way in these required fields, regardless of whether we're using in DART or not. Something to think about!

Reply:
Ah subclassing seems like a interesting idea